tg-portfolio/components/skills/Programming.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Link from 'next/link';
+import { urlFor } from '../../utils/client';
+
+const Programming = ({ technolos }) => (
+  <div className="flex md:flex-row place-content-start sm:flex-col justify-between scrollbar-hide ">
+    <div className="flex flex-col items-start text-green-700 font-bold animate-in zoom-in spin-in-90 duration-1000">
+      <div className="m-3"><h1>FRONT-END</h1> <progress className="progress md:w-96 sm:w-64" value="80" max="100" /></div>
+      <div className="m-3"><h1>BACK-END</h1><progress className="progress md:w-96 sm:w-64" value="20" max="100" /></div>
+      <div className="m-3"><h1>JAVASCRIPT</h1><progress className="progress md:w-96 sm:w-64" value="90" max="100" /></div>
+      <div className="m-3"><h1>PYTHON</h1><progress className="progress md:w-96 sm:w-64" value="10" max="100" /></div>
```

Comment:
Refactor the code by creating an array of your skills and map that data over the div structure

Reply:
Thank-you 😄